package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.0.2",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "gh": "ng build --prod --base-href \"https://initiafy.github.io/design-system-initiafy/\"",
-    "gh-pub": "npx ngh --dir=dist/InitiafyDesignSystem",
```

Comment:
lets keep this github pages stuff - this allows us to publish our site to https://initiafy.github.io/design-system-initiafy//

Reply:
sure, added
it wasn't in my last version that was working, sorry